deep-learning/_posts/2019-04-04-BreastCancerScreening.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: review
+title:  "Deep Neural Networks Improve Radiologists' Performance in Breast Cancer Screening"
+tags:   deep-learning medical-imaging
+author: Antoine Théberge
+pdf:    "https://arxiv.org/abs/1903.08297"
+cite:
+    authors: "Nan Wu, Jason Phang, Jungkyu Park, Yiqiu Shen, Zhe Huang, Masha Zorin, Stanisław Jastrzębski, Thibault Févry, Joe Katsnelson, Eric Kim, Stacey Wolfson, Ujas Parikh, Sushma Gaddam, Leng Leng Young Lin, Kara Ho, Joshua D. Weinstein, Beatriu Reig, Yiming Gao, Hildegard Toth, Kristine Pysarenko, Alana Lewin, Jiyon Lee, Krystal Airola, Eralda Mema, Stephanie Chung, Esther Hwang, Naziya Samreen, S. Gene Kim, Laura Heacock, Linda Moy, Kyunghyun Cho, Krzysztof J. Geras"
```

Comment:
May be name a few (5 or 6) and use _et al._ since otherwise the list produces a too long scroll due to the table layout?